test/unit/org/apache/cassandra/db/VerifyTest.java

```diff
@@ -758,6 +781,65 @@ public void testNoFilterFile()
         }
     }
 
+    @Test
+    public void testVerifyWithoutRealm()
+    {
+        CompactionManager.instance.disableAutoCompaction();
+        Keyspace keyspace = Keyspace.open(KEYSPACE);
+        ColumnFamilyStore cfs = keyspace.getColumnFamilyStore(CF);
+
+        fillCF(cfs, 2);
+
+        Descriptor descriptor = cfs.getLiveSSTables().iterator().next().getDescriptor();
+        SSTableReader sstable = descriptor.getFormat()
+                                          .getReaderFactory()
+                                          .openNoValidation(descriptor,
+                                                            SSTableReader.componentsFor(descriptor),
+                                                            TableMetadataRef.forOfflineTools(cfs.metadata()));
+
+        try (Verifier verifier = new Verifier(sstable,
+                                              true,
+                                              Verifier.options().invokeDiskFailurePolicy(true).build()))
+        {
+            verifier.verify();
+        }
+        catch (CorruptSSTableException err)
+        {
+            fail("Unexpected CorruptSSTableException");
+        }
+    }
+
+    @Test
+    public void testVerifyWithoutRealmMismatch()
```

Comment:
What's the intention of this test - what's the "mismatch"?
Probably I am missing something; from what I can see, this test differs from `testVerifyWithoutRealm` in that it sets a BigTable sstable format and there is a custom output handler for logging. Is it just testing BigTable format? Maybe I am expecting some error because of the "mismatch" in the test name?

Reply:
Good catch. I forgot to update the name when fixed the implementation.

Reply:
I improved the names